test/ctr.bats

```diff
@@ -711,3 +711,46 @@ function teardown() {
 	cleanup_pods
 	stop_crio
 }
+
+@test "ctr /etc/resolv.conf rw/ro mode" {
+	start_crio
+	run crioctl pod run --config "$TESTDATA"/sandbox_config.json
+	echo "$output"
+	[ "$status" -eq 0 ]
+	pod_id="$output"
+	writeconfig=$(cat "$TESTDATA"/container_config.json | python -c 'import json,sys;obj=json.load(sys.stdin);obj["command"] = ["sh", "-c", "echo test >> /etc/resolv.conf"]; json.dump(obj, sys.stdout)')
```

Comment:
Sorry to be a pain, but for now wouldn't having a couple more `container_config_*.json` files under testdata be more explicit about what we're trying to do ?
We already have quite a bit of duplication there, so we may eventually want to generate them once, at test time, from a few templates.